ksql-engine/src/main/java/io/confluent/ksql/function/udf/url/UrlDecodeParamKudf.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.url;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import io.confluent.ksql.function.KsqlFunctionException;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+
+
+@UdfDescription(
```

Comment:
I still think we _should_ include `Confluent` or `built in` as the author to differentiate from other UDF impls where the implementer does not provide an author.

Reply:
Thoughts on just setting the default author to be `built in` or `Confluent`?

Reply:
Other built in functions have `Confluent` as the author, so I'd go with that.  

As some point we should make them all use the same constant...

Reply:
but we already have some from non-confluent contributors in the code-base, what value should they have ? If we want folks to feel comfortable and willing to contribute we probably shouldn't "take ownership" of their functions this way (even though no-one intends that, perception being more important than reality sometimes and all that!). That's why i propose just leaving it empty - the whole `author` annotation seems primarily useful for an end-user organization trying to distinguish which team to talk to about a given function than anything else.

Reply:
I think I'm convinced by @blueedgenick's point. If we ever feel tempted to change this we can revisit it but for now I'll keep it blank.